paasta_tools/marathon_tools.py

```diff
@@ -227,7 +227,7 @@ def get_autoscaling_params(self):
         default_params = {
             'method': 'default',
         }
-        return self.config_dict.get('autoscaling', default_params)
```

Comment:
@solarkennedy not really since most python mutators don't return the result

The best you could do would be
```
result = {
    'method': 'default',
}
result.update(self.config_dict.get('autoscaling', {}))
return result
```